### DISCUSSION TITLE
UTF-16 not working in Jetbrains terminals.

### DISCUSSION THREAD

**Discussion by Ayfri** [JanDeDobbeleer/oh-my-posh#630] [Troubleshoot] [answered]
### Prerequisites

- [X] I have read and understand the `CONTRIBUTING` guide
- [X] I looked for duplicate issues before submitting this one

### Description

Hi, I'm using Windows with PowerShell 7, I installed this module, with the latest version, but I don't know why, UTF-16 icons are not working in Intellij/Webstorm terminals, so I have a lot of broken chars and ugly things...

### Environment

- Oh my Posh version: `3.133.1`
- Theme: `Custom one`_`(but it is also broken for all the default ones).`_
- Operating System: `Windows 10 (21H1 19043.899)`
- Shell: `PowerShell 7.1.1`
- Terminal: `Intellij/WebStorm`

**Comment by JanDeDobbeleer** [comment]
@Ayfri can you take a screenshot? Also, did you set a font in the Jetbrains terminal?

**Comment by Ayfri** [comment]
![image](https://user-images.githubusercontent.com/34159136/114274351-4a3adc80-9a1e-11eb-943d-86f9bdae5838.png)
I set the `Cascadia Code PL` font, the same as the New Terminal one.
_(Also, Icons that I used are weird because I was thinking that this plugin only support UTF-8 one, then I was searching for a globe, put the Unicode one, then find out that it was working on the New Terminal :/)_

**Comment by JanDeDobbeleer** [comment]
@Ayfri be aware that Cascadia Code PL does not contain all glyphs. So it's normal that for most themes that will contain rectangles.

**Comment by Ayfri** [comment]
Yes, I know, but as you see, the 🌐 icon doesn't work in the terminal of Webstorm, even though it's the same font :/

**Comment by JanDeDobbeleer** [comment, marked as answer]
@Ayfri so sorry, missed that. In that case it's a bug at Webstorm rather than OMP (we can't control the font).

**Comment by Ayfri** [comment]
Oh :/
I've made an issue to the Webstorm issue tracker so.

**Comment by JanDeDobbeleer** [comment]
@Ayfri can you add the link here?

**Comment by Ayfri** [comment]
[Here is the link.](https://youtrack.jetbrains.com/issue/IDEA-266533)
